imgparse/imgparse.py

```diff
@@ -662,5 +652,9 @@ def get_home_point(image_path, exif_data=None, xmp_data=None):
         else:
             raise KeyError()
     except KeyError:
-        logger.error("Couldn't parse home point. Sensor might not be supported")
-        raise ParsingError("Couldn't parse home point. Sensor might not be supported")
+        logger.warning(
```

Comment:
Should this be an error instead of a warning if an exception is raised? In similar places you don't log an error if an exception is raised, should this be removed entirely?

Reply:
This is an exception that is raised that is designed to be caught in `get_relative_altitude()`. Then logged message here is useful for letting the user know why the terrain elevation couldn't be determined when they request the relative altitude correct with `terrain` as a source